TUTORIAL.rst

```diff
@@ -0,0 +1,187 @@
+              acts as the model and does not get morphed. Hereinafter,
+              we will refer to the first PDF argument as the "objective"
+              and the second as the "reference", as the PDFmorph display
+              does.
+
+    6. Now, we will start the morphing process, which requires us to
+       provide initial guesses for our scaling factor, Gaussian smear,
+       and stretch, separately. We will start with the scaling factor.
+       Begin by typing the command ::
+
+            pdfmorph --scale=2 -a darkSub_rh20_C_01.gr darkSub_rh20_C_44.gr
+
+       Now, the difference Rw = 1.457, a significant increase from our
+       value previously. We must modify our initial value for the
+       scaling factor and do so until we see a reduction in the
+       difference Rw from the unmorphed value. Type ::
+
+            pdfmorph --scale=0.9 -a darkSub_rh20_C_01.gr darkSub_rh20_C_44.gr
+
+       The difference Rw is now 0.351, lower than our unmorphed
+       example's value. To see PDFmorph optimize the scale factor,
+       simply drop ``-a`` from the command and type ::
+
+            pdfmorph --scale=0.9 darkSub_rh20_C_01.gr darkSub_rh20_C_44.gr
+
+       PDFmorph, given a reasonable initial guess, will use find the
+       optimal value for each morphing feature. Here, we see that
+       PDFmorph displays ``scale = 0.799025`` in the command prompt,
+       meaning that it has found this to be the most optimal value for
+       the scale factor. The difference Rw = 0.330, indicating a
+       better fit than our reasonable initial guess.
+
+            * It is the choice of the user whether or not to run values
+              before removing ``-a`` when analyzing data with PDFmorph.
+              By including it, you allow the possibility to move towards
+              convergence before allowing the program to optimize by
+              removing it; when including it, you may reach a highly
+              optimized value on the first guess or diverge greatly.
+              In this tutorial, we will use it every time to check
+              for convergence.
+
+    7. Now, we will examine the Gaussian smearing factor. We provide an
+       initial guess by typing ::
+
+            pdfmorph --scale=0.8 --smear=0.5 -a darkSub_rh20_C_01.gr darkSub_rh20_C_44.gr
+
+       And viewing the results. We've tailored our scale factor to be
+       close to the value given by PDFmorph, but see that the difference
+       Rw has increased substantially due to our smear value. One
+       approach, as described above, is to remove the ``-a`` from the
+       above command and run it again.
+
+            * Note: The warnings that the Terminal/Command Prompt
+              displays are largely numerical in nature and do not
+              indicate a physically irrelevant guess. These are somewhat
+              superficial and in most cases can be ignored.
+
+       We see that this has had hardly any effect on our PDF. To see
```

Comment:
we can probably find a more interesting story for introducing this the rmin rmax values